[2021-08-12 15:30:23,573] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag_03.processing_tasks.jupyter_task.task_1 2021-08-12T13:29:08.614696+00:00 [queued]>
[2021-08-12 15:30:23,580] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag_03.processing_tasks.jupyter_task.task_1 2021-08-12T13:29:08.614696+00:00 [queued]>
[2021-08-12 15:30:23,581] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 15:30:23,581] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-12 15:30:23,581] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 15:30:23,595] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): processing_tasks.jupyter_task.task_1> on 2021-08-12T13:29:08.614696+00:00
[2021-08-12 15:30:23,600] {standard_task_runner.py:52} INFO - Started process 504584 to run task
[2021-08-12 15:30:23,608] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag_03', 'processing_tasks.jupyter_task.task_1', '2021-08-12T13:29:08.614696+00:00', '--job-id', '121', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/task_example.py', '--cfg-path', '/tmp/tmpbr00zlnz', '--error-file', '/tmp/tmpfza08dp9']
[2021-08-12 15:30:23,608] {standard_task_runner.py:77} INFO - Job 121: Subtask processing_tasks.jupyter_task.task_1
[2021-08-12 15:30:23,642] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag_03.processing_tasks.jupyter_task.task_1 2021-08-12T13:29:08.614696+00:00 [running]> on host adam
[2021-08-12 15:30:23,673] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag_03
AIRFLOW_CTX_TASK_ID=processing_tasks.jupyter_task.task_1
AIRFLOW_CTX_EXECUTION_DATE=2021-08-12T13:29:08.614696+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-12T13:29:08.614696+00:00
[2021-08-12 15:30:23,675] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-08-12 15:30:23,675] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo hello']
[2021-08-12 15:30:23,682] {subprocess.py:74} INFO - Output:
[2021-08-12 15:30:23,684] {subprocess.py:78} INFO - hello
[2021-08-12 15:30:23,685] {subprocess.py:82} INFO - Command exited with return code 0
[2021-08-12 15:30:23,702] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=parallel_dag_03, task_id=processing_tasks.jupyter_task.task_1, execution_date=20210812T132908, start_date=20210812T133023, end_date=20210812T133023
[2021-08-12 15:30:23,720] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-12 15:30:23,741] {local_task_job.py:149} INFO - Task exited with return code 0
